pkg/spdk/server.go

```diff
@@ -78,25 +78,29 @@ func (s *Server) monitoringReplicas() {
 	}
 }
 
-func (s *Server) verifyReplicas() error {
+func (s *Server) verify() error {
 	s.Lock()
 	defer s.Unlock()
 
-	bdevLvolList, err := s.spdkClient.BdevLvolGet("", 0)
+	bdevList, err := s.spdkClient.BdevGetBdevs("", 0)
 	if err != nil {
 		return err
 	}
 	subsystemList, err := s.spdkClient.NvmfGetSubsystems("", "")
 	if err != nil {
 		return err
 	}
+	bdevMap := map[string]*spdktypes.BdevInfo{}
 	bdevLvolMap := map[string]*spdktypes.BdevInfo{}
-	for idx := range bdevLvolList {
-		bdevLvol := &bdevLvolList[idx]
-		if len(bdevLvol.Aliases) == 0 {
-			continue
+	for idx := range bdevList {
+		bdev := &bdevList[idx]
+		if len(bdev.Aliases) == 1 && spdktypes.GetBdevType(bdev) == spdktypes.BdevTypeLvol {
+			lvolName := spdktypes.GetLvolNameFromAlias(bdev.Aliases[0])
```

Comment:
`bdev.Aliases[0]` is `test-block-1/pvc-39468503-106d-4606-9f1d-2f4879d188b5-r-0aaa43fd`
Then, `lvolName` is `pvc-39468503-106d-4606-9f1d-2f4879d188b5-r-0aaa43fd`.

Reply:
Yes. `lvolName` does not contain `lvsName`, and is actually replica name